### DISCUSSION TITLE
TP1-Problemas con if

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#76] [Preguntas y respuestas] [answered]
Hola, de a poco vamos entendiendo el lenguaje y pudimos avanzar con  @agosgit. Pero no nos funciona el if. Entendemos que la sintaxis está bien, pero preguntamos porque no nos funciona. 

Saludos

![ejercicio2](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/ce6c66c0-58a3-4162-bdd7-638f57d0951a)

**Comment by Nicolino07** [comment, marked as answer]
Hola. Les faltan las llaves en el if y else. Además el else va al mismo nivel de identacion con sus llaves

**Reply by Clirussa** [reply]
¡Muchas gracias!

**Comment by martinvilu** [comment]
Las llaves son lo que agrupan las instrucciones dentro de los bloques, y van siempre para evitar confusiones.

(no es como en Python que la agrupación la da la separación del margen)
